gradle.properties

```diff
@@ -11,6 +11,9 @@ doAnimalSniffer=false
 # The flag to include ORC support inside default h2o.jar.
 doIncludeOrc=false
 
+# The flag to include MOJO Pipeline support inside default h2o.jar.
+doIncludeMojoPipeline=false
```

Comment:
Do we even want such a option? It's a proprietary file. 

This might be really helpful for internal testing, but we need to be sure it does not leak anywhere.

Reply:
The dependency is only on mojo-runtime-api (just like it is included in SW I believe). @jakubhava do I get it right?

Reply:
I see :) The API is fine to use anywhere. I thought this means the implementation. All good then, thx!